src/material-examples/example-module.ts

```diff
@@ -17,10 +17,11 @@ import {AutocompleteDisplayExample} from './autocomplete-display/autocomplete-di
 import {AutocompleteFilterExample} from './autocomplete-filter/autocomplete-filter-example';
 import {AutocompleteOverviewExample} from './autocomplete-overview/autocomplete-overview-example';
 import {AutocompleteSimpleExample} from './autocomplete-simple/autocomplete-simple-example';
+import {ButtonCardExample} from './button-card/button-card-example';
```

Comment:
Not sure if you knew this but this file is auto-generated

(I'm not sure why it's in source control... @amcdnl?)